Data/Model/BaseModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Model
+{
+    public class BaseModel
+    {
+        public int Id { get; set; }
```

Comment:
long porque dependendo da tabela pode ter dados "infinitos"
```suggestion
        public long Id { get; set; }
```